ach/src/main/java/com/adyen/checkout/ach/AddressConfiguration.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by onurk on 24/1/2023.
+ */
+
+package com.adyen.checkout.ach
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Configuration class for Address Form in ACH Component. This class can be used define the
+ * visibility of the address form.
+ */
+sealed class AddressConfiguration : Parcelable {
```

Comment:
Should we call this `ACHDirectDebitAddressConfiguration` since it's specific to ACH only?

Reply:
It is specific to ACH. When I refactored the address, I changed the name from AddressConfiguration to CardAddressConfiguration. Özgür suggested keeping the AddressConfiguration naming for each component. I changed it, so I used the same pattern here.

Reply:
@ozgur00 adding you to the conversation.
 
Ok I understand the idea, and renaming it to `AddressConfiguration` to `CardAddressConfiguration` is an unnecessary breaking change. 

However I was thinking if someone has both configurations in the same file (like our `CheckoutConfigurationProvider`) they will need to use name aliases or full class name with package to use both classes.

What are your thoughts?

Reply:
That's a valid point. Initial point was indeed the unnecessary breaking change for existing users. In order to prevent the problem you mentioned we can rename these.

Reply:
Cool. Should we also rename `AddressConfiguration` to `CardAddressConfiguration`? It's an unnecessary breaking change but maybe it will look better 🤔

Reply:
Also @onurkaragunlu, can you use `ACHDirectDebitAddressConfiguration` please? 😅

Reply:
> Should we also rename `AddressConfiguration` to `CardAddressConfiguration`? It's an unnecessary breaking change but maybe it will look better 🤔

@ozgur00 this PR is good to go but just thinking about this, WDYT?